lib/ui/view/edit_profile/edit_profile_view.dart

```diff
@@ -98,47 +99,126 @@ class Body extends ViewModelWidget<EditProfileViewModel> {
+                    // TextFormField(
+                    //   initialValue: model.name,
+                    //   onChanged: (value) {
+                    //     model.updateString(value, '', '', '');
+                    //   },
+                    //   decoration: const InputDecoration(
+                    //     labelText: 'Full Name',
+                    //   ),
+                    // ),
                   )
                 ],
               ),
             ),
-            TextFormField(
-              initialValue: model.userData.displayName,
+            SizedBox(
+              height: 26,
+            ),
+            CustomTextField(
+              keyboardType: TextInputType.text,
+              inputAction: TextInputAction.next,
+              autoCorrect: false,
+              obscureText: false,
+              labelText: 'Display Name',
               onChanged: (value) {
                 model.updateString('', value, '', '');
               },
-              decoration: InputDecoration(
-                labelText: "Display Name",
-                helperText:
+              initialValue: model.userData.displayName,
+                            helperText:
                     "This is how your name will show up in Zuri Chat. It’s best kept simple: whatever people call you in everyday conversation.",
-                helperMaxLines: 3,
-              ),
+              helperMaxLines: 3,
             ),
-            TextFormField(
-              initialValue: model.userData.status,
+
+            // TextFormField(
+            //   initialValue: ''' Please open and close this '''
+            //       '''page twice to see changes after saving''',
+            //   onChanged: (value) {
+            //     model.updateString('', value, '', '');
+            //   },
+            //   decoration: const InputDecoration(
+            //     labelText: 'Display Name',
+            //     helperText: '''This is how your name will show '''
+            //         '''up in Zuri Chat. It’s best kept simple: whatever '''
+            //         '''people call you in everyday conversation.''',
+            //     helperMaxLines: 3,
+            //   ),
+            // ),
+            SizedBox(
+              height: 10,
+            ),
+
+
+                       CustomTextField(
+              keyboardType: TextInputType.text,
+              inputAction: TextInputAction.next,
+              autoCorrect: false,
+              obscureText: false,
+             labelText: 'What I do',
               onChanged: (value) {
                 model.updateString('', '', value, '');
               },
-              decoration: InputDecoration(
-                  labelText: "What I do", helperText: "HNGi9 X I4G"),
+              initialValue: model.userData.status,
+           helperText: 'HNGi9 X I4G'
+              
             ),
-            TextFormField(
-              initialValue: model.userData.phoneNum,
-              onChanged: (value) {
+
```

Comment:
delete unused code, or specify reason for keeping

Reply:
Done deleting the unused code